Code/GraphMol/FilterCatalog/FilterMatchers.h

```diff
@@ -350,12 +356,10 @@ class RDKIT_FILTERCATALOG_EXPORT SmartsMatcher : public FilterMatcherBase {
   template <class Archive>
   void load(Archive &ar, const unsigned int version) {
     ar &boost::serialization::base_object<FilterMatcherBase>(*this);
-    {
-      RDUNUSED_PARAM(version);
-      std::string res;
-      ar &res;
-      d_pattern = boost::shared_ptr<ROMol>(new ROMol(res));
-    }
+    RDUNUSED_PARAM(version);
+    std::string res;
+    ar &res;
```

Comment:
We should probably pick one of & versus << and >>, I, personally, am happy with & but it is less explicit.

Reply:
I don't really care, as long as we're consistent within any given file.

Reply:
i.e. I'm happy to change it if you want me to

Reply:
I'm happy with either and the use of RD_UNUSED so I say we merge.